src/spring/azext_spring/app.py

```diff
@@ -447,11 +456,67 @@ def _log_application(cmd, client, no_wait, poller, resource_group, service, app_
                               since=300, timeout=10, get_app_log=_get_app_log_deploy_phase)
     except Exception:
         # ignore
-        return
+        pass
     if deployment_error:
         raise deployment_error
 
 
+def _print_deploy_process(client, poller, resource_group, service, app_name, deployment_name):
+    try:
+        deployment_resource = _get_deployment_ignore_exception(client, resource_group, service, app_name,
+                                                               deployment_name)
+        if deployment_resource is not None:
+            instance_count = deployment_resource.sku.capacity
+            rolling_number = max(1, instance_count // 4)
+            rounds = int(instance_count // rolling_number + 0 if instance_count % rolling_number == 0 else 1)
+
+            if instance_count > 1:
+                instance_desc = str(instance_count) + " instances"
+                rounds_desc = str(rounds) + " rounds"
+            else:
+                instance_desc = str(instance_count) + " instance"
+                rounds_desc = str(rounds) + " round"
+            logger.warning('ASA will use rolling upgrade to update your deployment, you have {}, '
+                           'ASA will update the deployment in {}'.format(instance_desc, rounds_desc))
+            last_round = 0
+
+            deployment_time = datetime.datetime.now(tz=datetime.timezone.utc).strftime("%Y-%m-%dT%H:%M:%S%z")
```

Comment:
we should the current time. Because pod may restart due to oom/regular maintenace. 
if we use deployment.system_data.updated_at, we may wrongly treat some old pods as the new deployed.

Reply:
I am concern that some pod was already created but was thought as old pod

Reply:
The update_at should be the time that the PATCH happens

Reply:
fixed